### PR TITLE
[stable/k8s-spot-termination-handler] Call for maintainers for charts with a single OWNER

### PR DESCRIPTION
**Pull request by mariusv**
this PR is part of #13867 and #13869

Signed-off-by: Marius Voila <myself@mariusv.com>

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [ ] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

**Review by egeland — APPROVED** [review]

File: stable/k8s-spot-termination-handler/Chart.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: "1.13.7-1"
 description: The K8s Spot Termination handler handles draining AWS Spot Instances in response to termination requests.
 name: k8s-spot-termination-handler
-version: 1.4.3
+version: 1.4.4
```
Comment (egeland) [inline stable/k8s-spot-termination-handler/Chart.yaml]: Looks like you need to bump it
https://prow.k8s.io/view/gcs/kubernetes-jenkins/pr-logs/pull/helm_charts/16946/pull-charts-e2e/1173406394401951744#0:build-log.txt%3A181

**Review by egeland — CHANGES_REQUESTED** [review]

File: stable/k8s-spot-termination-handler/Chart.yaml (hunk 1 above)
Comment (mariusv) [reply stable/k8s-spot-termination-handler/Chart.yaml]: Looks like the Chart version changed after my PR. Should be fixed now

**Comment by maorfr** [comment]
added @mariusv as a trusted collaborator.

may i also ask that you rebase this PR before merging it? thanks!

File: stable/k8s-spot-termination-handler/Chart.yaml [hunk 2]
```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: "1.13.7-1"
 description: The K8s Spot Termination handler handles draining AWS Spot Instances in response to termination requests.
 name: k8s-spot-termination-handler
-version: 1.4.4
+version: 1.4.5
```
Comment (egeland) [inline stable/k8s-spot-termination-handler/Chart.yaml]: Current chart is at 1.4.6 now... 😅

**Review by egeland — CHANGES_REQUESTED** [review]

**Comment by maorfr** [comment]
/lgtm

welcome aboard!

**Pull request merged**